Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitStorageTypeOptionsForCluster.java

```diff
@@ -102,6 +111,29 @@
         this.optionTypes = optionTypes;
     }
 
+    SparkSubmitStorageTypeOptionsForCluster(StorageAccountTypeEnum type) {
+        switch(type){
+            case BLOB:
+            this.optionTypes = new SparkSubmitStorageType[]{
```

Comment:
coding style issue

Reply:
A tip in IntelliJ: We can select a piece of codes and apply `Ctrl + Alt + L` to auto-format the codes.

---
In reply to: [277987651](https://github.com/Microsoft/azure-tools-for-java/pull/3074#discussion_r277987651) [](ancestors = 277987651)